### DISCUSSION TITLE
Libadwaita

### DISCUSSION THREAD

**Discussion by alshdavid** [diamondburned/gotk4-examples#9] [Help] [answered]
Still new to GTK applications so please forgive me if I misunderstand how these pieces fit together.

My understanding (please correct me if I am wrong) is that Libadwaita is like a Widget library (also containing some functions and behaviours) to use with GTK.

The gotk4 bindings are only for GTK and do not expose libadwaita widgets/functions/interfaces, correct?

**Comment by diamondburned** [comment, marked as answer]
> My understanding (please correct me if I am wrong) is that Libadwaita is like a Widget library (also containing some functions and behaviours) to use with GTK.

Yes. More specifically, it supplies mobile-friendly adaptive widgets for GTK4
applications to be developed with mobile support as well. But it also enforecs
its own stylesheets on top.

> The gotk4 bindings are only for GTK and do not expose libadwaita widgets/functions/interfaces, correct?

Yes. [gotk4-adwaita](https://github.com/diamondburned/gotk4-adwaita) is available separately.
I also wrote [adaptive](https://github.com/diamondburned/adaptive) that is
supposed to be a Go alternative to libadwaita.
